db/migrate/20180205105808_rename_theme_id_to_topic_id.rb

```diff
@@ -0,0 +1,6 @@
+class RenameThemeIdToTopicId < ActiveRecord::Migration[5.1]
+  def change
+    rename_column :datasets, :theme_id, :topic_id
```

Comment:
In a production environment - where we want zero-downtime - you would have to create a new column, migrate the data over, and delete the old column.

https://blog.codeship.com/rails-migrations-zero-downtime/